src/platform_impl/web/web_sys/event.rs

```diff
@@ -5,10 +5,13 @@ use std::convert::TryInto;
 use web_sys::{HtmlCanvasElement, KeyboardEvent, MouseEvent, PointerEvent, WheelEvent};
 
 pub fn mouse_button(event: &MouseEvent) -> MouseButton {
+    // https://w3c.github.io/uievents/#dom-mouseevent-button
     match event.button() {
         0 => MouseButton::Left,
         1 => MouseButton::Middle,
         2 => MouseButton::Right,
+        3 => MouseButton::Back,
+        4 => MouseButton::Forward,
         i => MouseButton::Other((i - 3).try_into().expect("very large mouse button value")),
```

Comment:
This was done a very long time ago, but why is this subtracting 3 unlike other backends?
Can we fix this here while we are at it?

Reply:
I fixed this in https://github.com/rust-windowing/winit/pull/2838.